### DISCUSSION TITLE
Scan qr codes with external scanner

### DISCUSSION THREAD

**Discussion by HannesMorre** [inventree/InvenTree#5981] [Q&A] [answered]
What are the options to scan QR codes with external scanners? We have several scanners we would like to use:
- A wedge scanner connected to a computer
- A glove scanner that can be connected to an android device with the inventree app
- A smartphone with integrated scanner that has the inventree app installed on it

Can we use these external scanners or is this not supported yet?

**Comment by SchrodingersGat** [comment, marked as answer]
All three should be supported already:

1 - Supported in the web app
2 - Supported in the mobile app (enable wedge mode)
3 - Supported in the mobile app (enable wedge mode)

Ref: https://docs.inventree.org/en/latest/app/barcode/#barcode-input-methods
